docs/source/classes/Tutorials/1_Qubit/16a_JaynesCummingsHamiltonianBasis.ipynb

```diff
@@ -41,11 +41,11 @@
     "\n",
     "- $H_{JC} = \\hbar\\omega_{c} a^{\\dagger}a\\otimes 1_{2,2} + \\frac{1}{2}\\hbar\\omega_{q}1_{d,d}\\otimes\\sigma_{z} + \\hbar g(a^{\\dagger}\\otimes\\sigma_{-} + a\\otimes\\sigma_{+})$\n",
     "\n",
-    "    ![Cavity first and qubit with sigma-plus/minus operators and plus](CavityQubitWithSandPlus.png){ width=70% height=70% }\n",
```

Comment:
These (ie { width=70% height=70% } tags) might look like typos because they removing them does not change anything how the figures look like inside the notebook, but they take effect when we compile these notebooks to html and these are important to have. Could you please restore it back ? thanks.

Reply:
Fixed